contracts/ChildChainGaugeInjectorV2.sol

```diff
@@ -389,14 +389,17 @@ KeeperCompatibleInterface
     }
 /**
  * @notice Gets the total weekly spend
+ * @param includeFutureSchedules If true, the total spend will include future schedules, if false will only include schedules with start time before now
  * @return weeklySpend  The total amount of tokens required to fulfil all active programs for 1 period.
  */
-    function getWeeklySpend() public view returns (uint256 weeklySpend){
+    function getWeeklySpend(bool includeFutureSchedules) public view returns (uint256 weeklySpend){
         address[] memory gauges = getActiveGaugeList();
         for (uint256 i = 0; i < gauges.length; i++) {
             Target memory target = GaugeConfigs[gauges[i]];
             if (target.periodNumber < target.maxPeriods) {
-                weeklySpend += target.amountPerPeriod;
+                if(includeFutureSchedules || target.programStartTimestamp <= block.timestamp) {
+                    weeklySpend += target.amountPerPeriod;
+                }
             }
         }
         return weeklySpend;
```

Comment:
mostly informational comment

but when you are using an explicit returned value named like in this case `weeklySpend`. it is not neccesarily to also do a `return weeklySpend;` in L405

Reply:
Noted.  Trying really to not make more changes, but for V2.1 or whatnot and in future projects, good advice.